.travis.yml

```diff
@@ -25,19 +25,10 @@ install:
     - if [ "$CHECKS" = "yes" ]; then travis_retry composer require --no-interaction fabpot/php-cs-fixer:^1.11; fi;
```

Comment:
I basically agree, but it's a bit inconvenient for local development. For example, now I need to use my global installation of php-cs-fixer. Do you suggest that I install them globally?